learningequality/studio
# Prevent duplicate sync polling

## Summary
### Description of the change(s) you made
* Removes my broken vendored version of createLeaderElection and reverts to using updated version from Broadcast channel
* Handles duplicate leaders
* Removes no longer necessary change polling code

### Manual verification steps performed
1. Open two tabs
2. Confirm that backend requests are made only in one tab
3. Make a change in the other tab
4. Confirm that sync only happens in one tab, not both

### Does this introduce any tech-debt items?
No - removes tech debt by removing wonky vendored/modified code.
Original reason for vendoring should now be obsolete, and was motivated by unnecessary future planning for service workers.
___
## Reviewer guidance
### How can a reviewer test these changes?
See steps above.


### Are there any risky areas that deserve extra testing?
Make sure that removing the change polling isn't an issue - but we added it before we realized that `__` prefixed tables were not observed by Dexie Observable.


## References
Fixes #3191 

May also help to prevent #3210, #3164, #3163

----

### Contributor's Checklist


PR process:


Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
